openml/datasets/dataset.py

```diff
@@ -1061,14 +1114,3 @@ def _check_qualities(qualities: List[Dict[str, str]]) -> Dict[str, float]:
             value = float(xmlquality["oml:value"])
         qualities_[name] = value
     return qualities_
-
-
-def _parse_qualities_xml(qualities_xml):
```

Comment:
This looks like it is reverting some changes from @PGijsbers, is this on purpose or rather a merge issue?

Reply:
I don't see any reason to revert the changes at least.